make/CompileInterimLangtools.gmk

```diff
@@ -74,7 +74,7 @@ define SetupInterimModule
       EXCLUDE_FILES := $(TOPDIR)/src/$1/share/classes/module-info.java \
           Standard.java, \
       EXTRA_FILES := $(BUILDTOOLS_OUTPUTDIR)/gensrc/$1.interim/module-info.java, \
-      COPY := .gif .png .xml .css .js .txt javax.tools.JavaCompilerTool, \
+      COPY := .gif .png .xml .css .js .js.template .txt javax.tools.JavaCompilerTool, \
```

Comment:
Build-folk: it would be nice if this macro could use `$(jdk.javadoc.COPY)` instead of having to duplicate entries.
(Future RFE?)

Reply:
I agree. The entire design of CompileJavaModules.gmk needs to be updated. I've been procrastinating on cleaning this up, maybe it's time to get going on it...